app/views/service_worker/index.js.erb

```diff
@@ -187,6 +203,22 @@
             "/shell_bottom",
           ]));
       }
+
+      // Periodically delete assets when they pile up
```

Comment:
question: The bespoke periodic deletion of assets may not be necessary. I'm not positive, but see the [docs on Eviction](https://chromium.googlesource.com/chromium/src/+/master/content/browser/service_worker/README.md#Eviction) in the Service workers documentation.

Reply:
I think being explicit here might be safer to rely on any functionality we're not crystal clear on.

We also had some issues early on with eviction with service workers.

Reply:
Pulling this locally to give it a go.